nucypher/blockchain/eth/sol/source/contracts/WorkLock.sol

```diff
@@ -23,63 +23,92 @@ contract WorkLock {
     event Bid(address indexed sender, uint256 depositedETH);
     event Claimed(address indexed sender, uint256 claimedTokens);
     event Refund(address indexed sender, uint256 refundETH, uint256 completedWork);
-    event Burnt(address indexed sender, uint256 value);
     event Canceled(address indexed sender, uint256 value);
+    event BiddersChecked(address indexed sender, uint256 startIndex, uint256 endIndex);
+    event ClaimingEnabled(address indexed sender);
```

Comment:
I don't see any emits of this event in the code. Should it be deleted?

Reply:
Thanks, forgot to remove while rebasing, 
✔️